filebrowser/rockcraft.yaml

```diff
@@ -0,0 +1,158 @@
+# Developer note: See README.md for instructions on how to maintain this image and 
+# how builds theirs
+name: filebrowser
+base: ubuntu@22.04
+# using vX.Y.Z not X.Y.Z because upstream's convention includes the prefix "v"
+version: v2.27.0
+summary: An image for the filebrowser tool
+description: |
+  filebrowser is an interface for managing files, usable in a container
+  or as a standalone app.  See https://github.com/filebrowser/filebrowser
+  for more information.
+license: Apache-2.0
+run-user: _daemon_  # UID/GID 584792
+
+platforms:
+    amd64:
+
+services:
+  filebrowser:
+    override: merge
+    command: "/filebrowser [ ]"
+    working-dir: "/"
+    startup: enabled
+    on-check-failure:
+      alive: restart
+entrypoint-service: filebrowser
+
+checks:
+  alive:
+    override: replace
+    level: alive
+    period: 5s
+    exec:
+      command: /healthcheck.sh
+      working-dir: /
+      user: _daemon_
+      group: _daemon_
+
+parts:
+  security-team-requirement:
+    plugin: nil
+    override-build: |
+      mkdir -p ${CRAFT_PART_INSTALL}/usr/share/rocks
+      (echo "# os-release" && cat /etc/os-release && echo "# dpkg-query" && dpkg-query -f '${db:Status-Abbrev},${binary:Package},${Version},${source:Package},${Source:Version}\n' -W) > ${CRAFT_PART_INSTALL}/usr/share/rocks/dpkg.query
+
+  healthcheck:
+    plugin: nil
+    source: https://github.com/filebrowser/filebrowser.git
+    source-tag: v2.27.0
+    override-build: |
+      cp healthcheck.sh $CRAFT_PART_INSTALL/healthcheck.sh
+    stage-packages:
+      - curl
+      - jq
+    permissions:
+      - path: healthcheck.sh
+        # 584792 is the _daemon_ user
+        owner: 584792
+        group: 584792
+        mode: "755"
```

Comment:
definitely `healthcheck` and `add-config` could be combined, and probably also `create-srv-directory`.  To me I like them separate as it feels easier to debug, but I'm flexible.  

I don't know if there's a stylistic guide either.

Reply:
I 'm really torn on this cause I just see pros and cons on both sides. I 'll say let's keep it that way then, since we haven't standardized this. Should we add a note about style guide in the ROCKs meeting?

Reply:
+1

Reply:
I'm resolving this for now, but added a AI to the rock meeting